src/metatrain/utils/data/readers/readers.py

```diff
@@ -8,50 +9,75 @@
 from omegaconf import DictConfig
 
 from ..dataset import TargetInfo, TargetInfoDict
-from .systems import SYSTEM_READERS
-from .targets import ENERGY_READERS, FORCES_READERS, STRESS_READERS, VIRIAL_READERS
 
 
 logger = logging.getLogger(__name__)
 
+SUPPORTED_READER_LIBRARIES = ["ase"]
+""":py:class:`list`: list containing all implemented reader libraries"""
+
+READER_FORMATS = {
```

Comment:
Should this be called "DEFAULT_READERS"?

Reply:
I think you changed the wrong variable to "DEFAULT_READERS". IMO, this dict should be called DEFAULT_READERS:
```
{
    ".xyz": "ase",
    ".extxyz": "ase",
}
```
while this list
```
["ase"]
```
might be called "AVAILABLE_READERS" for example